Assets/Scripts/UI/CursorManagement.cs

```diff
@@ -0,0 +1,102 @@
+﻿using UnityEngine;
+
+public class CursorManagement : MonoBehaviour
+{
+    private const string CameraModeKey = "cameraType";
+
+    private static Mode PreferredMode = Mode.Original;
+    private static Mode CurrentMode = Mode.Menu;
+
+    public enum Mode : int
```

Comment:
@Lithrun prefers not specifying the underlying type, and he wants public enums to have their own file.